.studiorc

```diff
@@ -235,7 +235,7 @@ generate_bldr_keys() {
   keys=( $(find /hab/cache/keys -name "bldr-*.pub") )
 
   if [ "${#keys[@]}" -gt 0 ]; then
-    KEY_NAME=$(echo $keys[0] | grep -Po "bldr-\d+")
+    KEY_NAME=$(echo "${keys[0]}" | grep -Po "bldr-\d+")
```

Comment:
This syntax was just wrong. You always need braces to access an array variable:
```bash
$ arr=(one two three)
$ echo $arr[0]
one[0]
$ echo ${arr[0]}
one
```
The `-o` flag to `grep` just happened to save us.